packages/core/src/components/tabs/_tabs.scss

```diff
@@ -119,11 +124,6 @@ $tab-indicator-width: 3px !default;
   &:focus {
     -moz-outline-radius: 0;
   }
-
-  .pt-large & {
```

Comment:
why not `.pt-large > &`? i'd prefer that over moving `.pt-tab` styles inside the selector for a different element: keep the styles for one element inside that selector.